Atomic/util.py

```diff
@@ -794,6 +798,67 @@ def load_scan_result_file(file_name):
     """
     return json.loads(open(os.path.join(file_name), "r").read())
 
+
+class InstallData(object):
+
+    @staticmethod
+    def read_install_data():
+        if os.path.exists(ATOMIC_INSTALL_JSON):
+            return json.loads(open(os.path.join(ATOMIC_INSTALL_JSON), "r").read())
+        return {}
+
+    @staticmethod
+    def write_install_data(install_data):
+        with open(ATOMIC_INSTALL_JSON, 'w') as write_data:
+            time_out = 0
+            f_lock = False
```

Comment:
should the lock protect the entire (read + modify + write) operation?  Otherwise with two processes trying to write at the same time, we can lose what was written by the process that terminated first if they read the same initial file content.
An alternative can probably be to keep one separate file for each container, like:

`ATOMIC_VAR_LIB/install-data/${NAME}.json`

so we don't require any sort of synchronization.  Would it work?

Prune can cleanup the files that are left there without a related container still active (if the user manually uninstalled it, or the previous rm failed for any reason).